src/functions.h

```diff
@@ -32,11 +32,16 @@
 
 /*
  * functions.c unit provides the Redis Functions API:
- * * FUNCTION CREATE
- * * FUNCTION CALL
+ * * FUNCTION LOAD
+ * * FUNCTION LIST
+ * * FUNCTION CALL (FCALL and FCALL_RO)
  * * FUNCTION DELETE
+ * * FUNCTION STATS
  * * FUNCTION KILL
- * * FUNCTION INFO
```

Comment:
function create was renamed to function load. function info has been removed.

Another option is to delete this comment, otherwise we will need to modify it every time we add a command in the future.